packages/x-codemod/src/v7.0.0/data-grid/remove-stabilized-experimentalFeatures/remove-stabilized-experimentalFeatures.test.ts

```diff
@@ -24,4 +24,27 @@ describe('v7.0.0/data-grid', () => {
       expect(actual).to.equal(expected, 'The transformed version should be correct');
     });
   });
+
+  describe('remove-stabilized-experimentalFeatures - with typescript', () => {
```

Comment:
maybe this be merged with the previous `describe` and just include additional tests there